.github/workflows/Build.yml

```diff
@@ -0,0 +1,77 @@
+# ------------------------------------------------------------------------------
+# <auto-generated>
+#
+#     This code was generated.
+#
+#     - To turn off auto-generation set:
+#
+#         [CustomGitHubActions (AutoGenerate = false)]
+#
+#     - To trigger manual generation invoke:
+#
+#         nuke --generate-configuration GitHubActions_Build --host GitHubActions
+#
+# </auto-generated>
+# ------------------------------------------------------------------------------
+
+name: Build
+
+on:
+  push:
+    branches:
+      - master
+      - dev
+    paths:
+      - '!docs/**/*'
+      - '!package.json'
+      - '!README.md'
+  pull_request:
+    branches:
+      - master
+      - dev
+
+jobs:
+  windows-latest:
+    name: windows-latest
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 5.0.*
+      - uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 6.0.*
+      - name: Cache .nuke/temp, ~/.nuget/packages
+        uses: actions/cache@v2
+        with:
+          path: |
+            .nuke/temp
+            ~/.nuget/packages
+          key: ${{ runner.os }}-${{ hashFiles('global.json', 'src/**/*.csproj') }}
+      - name: Run './build.cmd Compile'
+        run: ./build.cmd Compile
+        env:
+          GITHUB_CONTEXT: ${{ toJSON(github) }}
+  ubuntu-latest:
+    name: ubuntu-latest
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 5.0.*
+      - uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 6.0.*
+      - name: Cache .nuke/temp, ~/.nuget/packages
+        uses: actions/cache@v2
+        with:
+          path: |
+            .nuke/temp
+            ~/.nuget/packages
+          key: ${{ runner.os }}-${{ hashFiles('global.json', 'src/**/*.csproj') }}
+      - name: Run './build.cmd Compile'
+        run: ./build.cmd Compile
+        env:
+          GITHUB_CONTEXT: ${{ toJSON(github) }}
```

Comment:
This was auto-generated by launching the nuke build console app once. To update, the console app needs to be launched again